src/network/taraxa_capability.cpp

```diff
@@ -1110,16 +1105,9 @@ void TaraxaCapability::sendPbftVote(NodeID const &_id, taraxa::Vote const &vote)
 }
 
 void TaraxaCapability::onNewPbftBlock(taraxa::PbftBlock const &pbft_block) {
-  std::vector<NodeID> peers_to_send;
+  std::vector<NodeID> peers_to_send = getAllPeers();
   auto my_chain_size = pbft_chain_->getPbftChainSize();
-  {
-    boost::shared_lock<boost::shared_mutex> lock(peers_mutex_);
-    for (auto const &peer : peers_) {
-      if (!peer.second->isPbftBlockKnown(pbft_block.getBlockHash())) {
```

Comment:
I guess you know what you're doing discarding this condition

Reply:
My mistake